topics/introduction/tutorials/galaxy-intro-short/tutorial.md

```diff
@@ -263,6 +265,39 @@ We have now decided that our input reads have to be filtered to an even higher s
 
 You can re-run a tool many times with different settings. Each time you re-run the tool, its new output datasets will appear at the top of your current history.
 
+
+## Convert your analysis history into a workflow
+
+When you look carefully at your history, you can see that it contains all steps of our analysis, from the beginning to the end. By building this history we have actually built a complete record of our analysis with Galaxy preserving all parameter settings applied at every step. But when you need to analyze new data, it would be tedious to do each step over again. Wouldn't it be nice to just convert this history into a workflow that we will be able to execute again and again?
+
+Galaxy makes this very easy with the `Extract workflow` option. This means any time you want to build a workflow, you can just perform the steps once manually, and then convert it to a workflow, so that next time it will be a lot less work to do the same analysis.
+
+> ### {% icon hands_on %} Hands-on: Extract workflow
+>
+> 1. **Clean up** your history.
+>    - If you had any failed jobs (red), please remove those datasets from your history by clicking on the `x` button. This will make the creation of a workflow easier.
```

Comment:
```suggestion
>    - If you had any failed jobs (red), please remove those datasets from your history by clicking on the {% icon galaxy-cross %} button. This will make the creation of a workflow easier.
```